crate.go

```diff
@@ -191,6 +193,34 @@ func (r *Rows) Close() error {
 	return nil
 }
 
+var typeMap = map[int]string{
+	2: "byte",
```

Comment:
Just one question, Go's documentation specifies that types should be returned uppercased, here:

https://golang.org/pkg/database/sql/driver/#RowsColumnTypeDatabaseTypeName

Also here is the discussion about this feature.

https://github.com/golang/go/issues/16652

Crate documents all of their types lowercased, I see no reason for us to uppercase them.

Before merging this, I would like to know if you have any thoughts on this.

Reply:
hello? @herenow 
I didn't know there were a spec for upper/lowercase.
I just used lowercase cause crate's document was written in lowercase and it's easy to type.
I thought about this for a while and looked for other golang drivers implementations.
Other drivers also already knows about this and implemented in uppercase, even commented it should be uppercase.
So I think uppercase would be better, cause somebody who knows about this spec, may expect that function will return uppercased.
And others my not cares about it. : )
Thanks you for asking my opinion.